wagtail/admin/views/generic/models.py

```diff
@@ -327,17 +319,48 @@ def get_add_url(self):
             return reverse(self.add_url_name)
 
     def get_valid_orderings(self):
-        orderings = []
+        orderings = set()
+        # Gather valid orderings from the Column instances
         for col in self.columns:
             if col.sort_key:
-                orderings.append(col.sort_key)
-                orderings.append("-%s" % col.sort_key)
+                orderings |= {col.sort_key, f"-{col.sort_key}"}
+
+        # Combine with any explicitly defined orderings
+        default_ordering = self.ordering or self.default_ordering
+        if default_ordering:
+            if isinstance(default_ordering, str):
+                default_ordering = (default_ordering,)
+            orderings |= set(default_ordering)
         return orderings
 
     def get_ordering(self):
-        ordering = self.request.GET.get("ordering", self.default_ordering)
+        default_ordering = self.ordering or self.default_ordering
+        ordering = self.request.GET.get("ordering")
         if ordering not in self.get_valid_orderings():
-            ordering = self.default_ordering
+            ordering = default_ordering
+        return ordering
+
+    def _get_ordering(self):
+        ordering = self.get_ordering()
+        if not ordering:
+            return ()
+
+        # Normalise to a tuple
+        if not isinstance(ordering, (list, tuple)):
+            ordering = (ordering,)
+
+        # Explicitly handle null values for the updated at column to ensure consistency
+        # across database backends and match the behaviour in page explorer
+        updated_at_asc = models.F("_updated_at").asc(nulls_first=True)
+        updated_at_desc = models.F("_updated_at").desc(nulls_last=True)
+        ordering = (
+            updated_at_asc
+            if order == "_updated_at"
+            else updated_at_desc
+            if order == "-_updated_at"
+            else order
+            for order in ordering
+        )
```

Comment:
The logic has been updated to handle the ordering as a list/tuple to make sure we handle it if multiple ordering fields have been specified (either by the default ordering, or multiple `GET` parameters, even though the latter isn't officially supported via the UI). The previous logic (deleted in `get_queryset()` above) only handles the case where only a single ordering field is specified.